spec/e2e/id_property_spec.rb

```diff
@@ -64,7 +64,7 @@
           Neo4j::Config[:id_property] = :the_id
           Neo4j::Config[:id_property_type] = :auto
           Neo4j::Config[:id_property_type_value] = :uuid
-          stub_const('Clazz', Class.new do
+          stub_const('Clazz', UniqueClass.create do
```

Comment:
Removing all constraints before each spec makes several spec fail as they meanwhile expect the bleeding over. I think this has to be fixed, but it is somehow our of the context of this pull request,